src/Modal/README.md

```diff
@@ -8,25 +15,7 @@ import { Modal } from '@tillersystems/stardust';
 
 <!-- STORY -->
 
-### Properties
-
-- `active` - Boolean set to display or hide the modal.
-- `children` - Anything that can be rendered: numbers, strings, elements or an array (or fragment).
-- `className` - className needed by styled components.
-- `height` - Modal height.
-- `width` - Modal width.
-- `padding` - Modal padding.
-- `onOverlayClick` - A model can have a clickable overlay to close it.
-
-| propName         | propType | defaultValue | isRequired |
-| ---------------- | -------- | ------------ | ---------- |
-| `active`         | `bool`   | `false`      | -          |
-| `children`       | `node`   | `null`       | -          |
-| `className`      | `string` | `''`         | -          |
-| `height`         | `string` | `48rem`      | -          |
-| `width`          | `string` | `39rem`      | -          |
-| `padding`        | `string` | `1.5rem`     | -          |
-| `onOverlayClick` | `func`   | `null`       | -          |
+<!-- PROPS -->
 
 ### Example
 
```

Comment:
Change the import here.
<img width="897" alt="Screen Shot 2019-08-21 at 14 51 56" src="https://user-images.githubusercontent.com/16764987/63433340-49a59a80-c423-11e9-945b-17f2a68305db.png">